src/lib/viewers/controls/annotations/annotations-button-region-tooltips/PersistentOnboardingBoxEditAnnotationsTooltip.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useContext, PropsWithChildren } from 'react';
+import classNames from 'classnames';
+
+// TODO: Compare bundle sizes and possibly re-write here
+import { TargetedClickThroughGuideTooltip } from 'box-ui-elements/es/features/targeting';
+
+import { currentLocale } from '../../../../i18n';
+
+import { TargetingApi } from '../../../../types';
+
+import { ControlsLayerContext } from '../../controls-layer';
+import { ExperiencesContext } from '../../experiences';
+
+import './PersistentOnboardingBoxEditAnnotationsTooltip.scss';
+
+const LOCALE_LANGUAGE_ENGLISH = 'en';
+
+export type Props = PropsWithChildren<{
+    isEnabled?: boolean;
+}>;
+
+export default function PersistentOnboardingBoxEditAnnotationsTooltip({
+    children,
+    isEnabled = false,
+}: Props): JSX.Element {
+    const { experiences } = useContext(ExperiencesContext);
+    const { setIsForced } = useContext(ControlsLayerContext);
+
+    const {
+        persistentOnboardingBoxEditAnnotations: experience = {
+            canShow: false,
+            onClose(): void {
+                // do nothing
+            },
+            onComplete(): void {
+                // do nothing
+            },
+            onShow(): void {
+                // do nothing
+            },
+            onPrevious(): void {
+                // do nothing
+            },
+        },
+    } = experiences;
+
+    const useTargetingApi = (): TargetingApi => ({
+        canShow: experience.canShow,
+        onClose: (): void => {
+            experience.onClose();
+            setIsForced(false);
+        },
+        onComplete: (): void => {
+            experience.onComplete();
+            setIsForced(false);
+        },
+        onShow: (): void => {
+            experience.onShow();
+            setIsForced(true);
+        },
+    });
+
+    const title = __('persistent_onboarding_box_edit_annotations_tooltip_title');
+    const body = __('persistent_onboarding_box_edit_annotations_tooltip_body');
+
+    const isLocaleLanguageEnglish = currentLocale === LOCALE_LANGUAGE_ENGLISH;
+    const image = (
+        <div
+            className={classNames('bp-PersistentOnboardingBoxEditAnnotationsTooltip-image', {
+                'is-localized': !isLocaleLanguageEnglish,
+            })}
+            data-testid="tooltipImage"
+        />
+    );
+
+    const primaryButtonProps = {
+        children: __('persistent_onboarding_box_edit_annotations_tooltip_button_next'),
+    };
+
+    const backButtonProps = {
```

Comment:
Is this still needed? It seems like the `TargetedClickThroughGuideTooltip` component should handle it.

Reply:
It is needed to declare the back button and bind the experience's `onPrevious()` callback to it.